depends/packages/libnatpmp.mk

```
@@ -3,20 +3,22 @@ $(package)_version=07004b97cf691774efebe70404cf22201e4d330d
 $(package)_download_path=https://github.com/miniupnp/libnatpmp/archive
 $(package)_file_name=$($(package)_version).tar.gz
 $(package)_sha256_hash=9321953ceb39d07c25463e266e50d0ae7b64676bb3a986d932b18881ed94f1fb
+$(package)_build_subdir=build
 
 define $(package)_set_vars
-  $(package)_build_opts=CC="$($(package)_cc)"
-  $(package)_build_opts_mingw32=CPPFLAGS=-DNATPMP_STATICLIB
-  $(package)_build_opts_darwin=LIBTOOL="$($(package)_libtool)"
-  $(package)_build_env+=CFLAGS="$($(package)_cflags) $($(package)_cppflags)" AR="$($(package)_ar)"
+  $(package)_config_opts += -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+endef
+
+define $(package)_config_cmds
+  $($(package)_cmake) -S .. -B .
 endef
 
 define $(package)_build_cmds
-  $(MAKE) libnatpmp.a $($(package)_build_opts)
+  $(MAKE) natpmp
 endef
 
 define $(package)_stage_cmds
   mkdir -p $($(package)_staging_prefix_dir)/include $($(package)_staging_prefix_dir)/lib &&\
-  install *.h $($(package)_staging_prefix_dir)/include &&\
+  install ../*.h $($(package)_staging_prefix_dir)/include &&\
```

nit: Maybe skip headers that are used only to build the package:
```suggestion
  install ../natpmp.h  ../natpmp_declspec.h $($(package)_staging_prefix_dir)/include &&\
```
?